### PR TITLE
PLAT-845: Text component

### PR DESCRIPTION
**Pull request by chavlji**
[JIRA ticket](https://reciprocitylabs.atlassian.net/browse/PLAT-845)

Check story Typography - Text:
Screenshot 2021-01-22 at 16 49 28

### CODE REVIEW

File: src/components/zen-text/zen-text.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  @Prop({ reflect: true }) readonly underline = false;
+
+  /** Underlined */
+  @Prop({ reflect: true }) readonly align: Align = 'left';
+
+  /** truncate */
+  @Prop({ reflect: true }) readonly truncate = false;
+
+  /** pale */
+  @Prop({ reflect: true }) readonly pale = false;
```
Comment (f-jauregui) [inline src/components/zen-text/zen-text.tsx:44]: Please fix these JSDoc comments.

**Review by f-jauregui — APPROVED** [review]
Good work! 👍

File: src/components/zen-text/zen-text.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  /** Layout as inlined */
+  @Prop({ reflect: true }) readonly inline = false;
+
+  /** Strikethrough */
+  @Prop({ reflect: true }) readonly strikethrough = false;
+
+  /** Underlined */
+  @Prop({ reflect: true }) readonly underline = false;
+
+  /** Underlined */
```
Comment (mplesec) [inline src/components/zen-text/zen-text.tsx:37]: `Underlined` is twice here.

**Review by mplesec — APPROVED** [review]
Awesome 👍

**Pull request merged**